Sources/APIInterface/include/WordPressComRESTAPIInterfacing.h

```diff
@@ -0,0 +1,15 @@
+@import Foundation;
+
+@protocol WordPressComRESTAPIInterfacing
+
+- (void)get:(NSString * _Nonnull)URLString
+ parameters:(NSDictionary<NSString *, NSObject *> * _Nullable)parameters
+    success:(void (^ _Nonnull)(id _Nonnull, NSHTTPURLResponse * _Nullable))success
+    failure:(void (^ _Nonnull)(NSError * _Nonnull, NSHTTPURLResponse * _Nullable))failure;
+
+- (void)post:(NSString * _Nonnull)URLString
```

Comment:
I decided not to uppercase `GET` and `POST` like elsewhere in the code because of the naming convention of starting methods names in lowercase.

Interestingly, even if uppercase, the conversion from Objective-C to Swift results in the word being lowercased in the generated Swift version.

Reply:
Also interesting, Xcode suggested the method interface. I think it's a new Xcode 15.2/15.3 feature? I was pleasantly surprised.